src/test/java/com/example/employee/service/EmployeeServiceTest.java

```diff
@@ -0,0 +1,81 @@
+package com.example.employee.service;
+
+import com.example.employee.model.Employee;
+import com.example.employee.model.EmployeeDTO;
+import com.example.employee.model.EmployeeMapper;
+import com.example.employee.repository.EmployeeRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.times;
+
+@ExtendWith(MockitoExtension.class)
+class EmployeeServiceTest {
+
+    @Mock
+    EmployeeRepository employeeRepository;
+    EmployeeMapper employeeMapper;
+    EmployeeService employeeService;
+
+    @BeforeEach
+    void setUp(){
+        employeeMapper = new EmployeeMapper();
+        employeeService = new EmployeeService(employeeRepository, employeeMapper);
+    }
+
+    @Test
+    void createAndUpdateEmployee() {
+        EmployeeDTO employeeDTO = new EmployeeDTO(1L, "Ilia","Tchavtchavadze", "Education", "Founder of education council", "ilia.tchavtchavadze@gmail.com", true, true);
+        Mockito.when(employeeRepository.save(any())).thenAnswer( invocationOnMock -> invocationOnMock.getArgument(0));
+
+        EmployeeDTO actualDTO = employeeService.createAndUpdateEmployee(employeeDTO);
+
+        Assertions.assertEquals(employeeDTO, actualDTO);
+        System.out.println("Create and Update test passed successfully!");
+    }
+
+    @Test
+    void findAllEmployees() {
+        Employee emp1 = new Employee(5L,"Nika","Avalishvili","D1","P1","as@gmail.com",true, true);
+        Employee emp2 = new Employee(6L,"Giorgi","Margvelashvili","D2","P2","as@gmail.com",true, true);
+        Mockito.when(employeeRepository.findAll()).thenReturn(Stream.of(emp1,emp2).collect(Collectors.toList()));
```

Comment:
We can replace this `Stream.of(emp1,emp2).collect(Collectors.toList())` with just `List.of(emp1,emp2)`